projects/fsastorefrontlib/src/cms-components/my-account/policy/policies-chart/policies-chart.component.ts

```diff
@@ -18,7 +18,7 @@ import { Config, LanguageService, TranslationService } from '@spartacus/core';
 export class PoliciesChartComponent implements OnInit, OnDestroy {
   chartOption: EChartsOption;
   options: any[];
-  selectedFrequency: string;
+  selectedFrequency: any;
   policiesByPaymentFrequency;
```

Comment:
Should be a type of `Policy`?

Reply:
No, since it is an object of key(string) value(policies array) pair.

Reply:
I figured since we are returning `policy` in `groupPoliciesByAttribute` method, maybe it should be a type of Policy. I see now that we don't have Policy as an interface...